dbschema/location.esdl

```diff
@@ -0,0 +1,28 @@
+module default {
+  abstract type Location extending Resource {
+    required name: str {
+      constraint exclusive;
+    }
+    required type: Location::Type;
+
+    isoAlpha3: Ethnologue::code {
```

Comment:
Hmm yeah this is a 3 digit alpha lower cased code....I hate to reference ethnologue here though. Two unrelated standards. I think I'd be fine copy pasting that here to declare this as an `IsoAlpha3Code`.
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
Wikipedia actually uppercases them...Oh actually so do we. So it's a unique regex anyways.

Reply:
That's what I was getting at.  It wasn't exactly the same regex.

Reply:
Yep! My b if I miscommunicated that earlier today